src/tests/align.test.tsx

```diff
@@ -60,7 +60,7 @@ describe("aligning", () => {
     ReactDOM.unmountComponentAtNode(document.getElementById("root")!);
     mouse.reset();
 
-    await setLanguage("en.json");
+    await setLanguage(languages[0]);
```

Comment:
Let's export `defaultLanguage` from `i18n` so we don't have to do `languages[0]` everywhere. That's error prone.